### DISCUSSION TITLE
Stable Version

### DISCUSSION THREAD

**Discussion by IamRezaMousavi** [fzyzcjy/flutter_rust_bridge#2060] [General]
When flutter_rust_bridge become to stable version?
I look forward to the stable version :)

**Comment by fzyzcjy** [comment]
Hi, as is mentioned in README (`What's new in V2` - click "expand")

> I want to keep it in beta for a while (though CI has all passed), to allow publishing API breaking changes, and hear your thoughts and suggestions about it!

Yes I will publish stable version later, but cannot guarantee an exact date.

On the other hand, IMHO the beta version looks quite stable (there are issues in the issue tracker, but many are feature requests, how to use something, etc, and there do exists bugs but most of them are not big). And if you see any bugs, feel free to open an issue, and I usually reply quickly!

In short, I personally suggest using 2.0 beta now, though it has a name "beta" - but that is mostly to allow me to have (rare) breaking changes.

**Reply by fzyzcjy** [reply]
@IamRezaMousavi btw stable version is released

**Reply by IamRezaMousavi** [reply]
> @IamRezaMousavi btw stable version is released

Thank you :))

**Reply by fzyzcjy** [reply]
You are welcome!
